src/uct/ib/configure.m4

```diff
@@ -19,7 +19,19 @@ AS_IF([test -d "$with_verbs/lib64"],[libsuff="64"],[libsuff=""])
 
 AC_MSG_NOTICE([Compiling $str])
 
+
+#
+# MLX5 DV Support, Provides RC/UD accelerated, DC and DV transports
```

Comment:
DV transports sounds confusing, maybe
```suggestion
# MLX5 DV Support, Provides RC/UD accelerated and DC transports
```

Reply:
IMO "provides accelerated RC/UD/DC transports"  would be ok

Reply:
fixed